spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe ApplicationHelper do
+  describe '.featured_collections' do
+    it 'should return the output of FeaturedItems.collections' do
+      expect(helper.featured_collections).to eq(FeaturedItems.collections)
```

Comment:
testing what is being returned is good but we should also test if that method is being called.

more info about how to do it: https://relishapp.com/rspec/rspec-mocks/docs/basics/expecting-messages